service/frontend/workflow_handler.go

```diff
@@ -4121,6 +4113,46 @@ func (wh *WorkflowHandler) validateWorkflowIdReusePolicy(
 	return nil
 }
 
+func (wh *WorkflowHandler) validateWorkflowCompletionCallbacks(
+	ns namespace.Name,
+	callbacks []*commonpb.Callback,
+) error {
+	if len(callbacks) > 0 && !wh.config.EnableCallbackAttachment(ns.String()) {
+		return status.Error(
+			codes.InvalidArgument,
+			"attaching workflow callbacks is disabled for this namespace",
+		)
+	}
+
+	for _, callback := range callbacks {
+		switch cb := callback.GetVariant().(type) {
+		case *commonpb.Callback_Nexus_:
+			if len(cb.Nexus.GetUrl()) > wh.config.CallbackURLMaxLength() {
```

Comment:
nit: If cb.Nexus isn't nil you could use `cb.Nexus.Url` directly

Reply:
cc: @bergundy

Reply:
I think the `type` ensures that `cb.Nexus` isn't nil but I'm not 100% sure.